src/Compiler/Microsoft.CodeAnalysis.Razor.Compiler/src/CSharp/CompilationExtensions.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.CodeAnalysis.Razor.Compiler.CSharp;
+internal static class CompilationExtensions
```

Comment:
nit: Please add a blank line between the namespace and class declaration.

Reply:
ugh, new file not doing that is definitely a frustration